user_manual/pim/sync_thunderbird.rst

```diff
@@ -62,4 +62,4 @@ https://cloud.nextcloud.com/remote.php/dav/calendars/daniel/personal/
 
 .. image:: ../images/CalDAV_calendar.png
 
-The rest of the details of dealing with Thunderbird Calendar are left to the reader... 
+The rest of the details of dealing with Thunderbird Calendar are left to the reader...
```

Comment:
This sentence seems kind of weird to me. It kind of sounds like "We were to lazy to explain more. Go figure it out yourself, not our problem."

We explained how to connect Lightning to Nextcloud, so I would just replace this sentence with something like:
"Further instructions how to manage your calendars with Thunderbird Lightning can be found in their respecting documentation: https://wiki.mozilla.org/Calendar:User_Documentation"

Same goes for Cardbook above.

Reply:
=> Another PR